proto-lens-tests/tests/enum_test.hs

```diff
@@ -163,3 +165,5 @@ testAliases = testCase "alias" $ do
 
 testManyCases =
     runTypedTest (roundTripTest "many cases" :: TypedTest ManyCasesProto)
+
+testLone = testCase "no messages" $ LONE @?= LONE
```

Comment:
I'm not sure I understand this test. Isn't `LONE` always `== LONE`?

Reply:
Added a comment.  The test is intentionally trivial; the goal is just to make this file import the other module to make sure it compiles properly.

Reply:
Got it. Thanks for the comment.